### DISCUSSION TITLE
Chat - Priority.

### DISCUSSION THREAD

**Discussion by Aides123** [Zrips/CMI#6399] [Enhancement]
Hey!
I have a small problem with the CHAT module.

I'm using "FunnyGuilds" plugin which adds private chat for team, allies. However, the problem is that when a player tries to use this chat via the !text shortcut he gets sent to the global and guild chat. Is it possible to add weight/priority for CMI chat?

Message with !1231 she should not appear, but the one after her
![image](https://user-images.githubusercontent.com/23367220/143607866-d1b26577-66d3-42f9-8e30-5f2e23d3baf6.png)

**Comment by ayushgameretawah** [comment]
i want to show cmi ranks in tab and chat both so how to do that please tell

**Reply by mrfloris** [reply]
> i want to show cmi ranks in tab and chat both so how to do that please tell

Using CMI placeholders, https://www.zrips.net/cmi/placeholders/

**Reply by ayushgameretawah** [reply]
can you explain step by step i cant under stand in this website
